magic_reservoir/README.md

```diff
@@ -0,0 +1,2 @@
+### Jupyter notebook to create plot of streamflow above and below Magic Reservoir
+* https://github.com/USGS-VIZLAB/viz-scratch/blob/magic_reservoir/magic_reservoir/Magic.ipynb
```

Comment:
Since it is `blob/branch_name`, you should change to `https://github.com/USGS-VIZLAB/viz-scratch/blob/master/magic_reservoir/Magic.ipynb` so that it is correct when merged.